### PR TITLE
20230320 resolve arweave links

### PR DESCRIPTION
**Pull request by vadim-frolov**

### CODE REVIEW

File: bin/node/cli/src/main.rs [hunk 1]
```diff
@@ -124,6 +124,10 @@ async fn main() {
             "https".to_string(),
             Box::new(HTTPSLinkResolver::new().expect("failed to create HTTP client")),
         )
+        .with_link_resolver(
+            "ar".to_string(),
```
Comment (jorgeantonio21) [inline bin/node/cli/src/main.rs:128]: nit: arweave ?

**Review by jorgeantonio21 — APPROVED** [review]
LGTM !

File: bin/node/cli/src/main.rs (hunk 1 above)
Comment (vadim-frolov) [reply bin/node/cli/src/main.rs:128]: I used the URI protocol for now ("https", "ipfs", "ar", etc.). We can discuss.

**Comment by benjaminbollen** [comment]
LGTM

**Pull request merged**